_includes/selector.html

```diff
@@ -288,7 +288,7 @@
                 var indent = "    ";
                 var cmd = this.highlightCmd("docker") + " pull " + fullImg + "\n" +
                     this.highlightCmd("docker") + " run --" + this.highlightFlag("gpus") + " all --" + this.highlightFlag("rm") + " -" + this.highlightFlag("it") + " \\\n" +
-                    indent + "--" + this.highlightFlag("shm-size") + "=1g --" + this.highlightFlag("ulimit") + " memlock=-1" + " \\\n" +
+                    indent + "--" + this.highlightFlag("shm-size") + "=1g --" + this.highlightFlag("ulimit") + " memlock=-1 --" + this.highlightFlag("ulimit") + " stack=67108864" + " \\\n" +
```

Comment:
@taureandyernv Where does this magic value of $2^{26}$ come from? Is it just "a large number" or does it mean something?